packages/shared/src/components/post/PostContent.tsx

```diff
@@ -89,7 +89,7 @@ const PageBodyContainer = classed(
 
 const PostContainer = classed(
   'main',
-  'flex flex-col flex-1 px-8 pb-20 laptop:max-w-[40rem] tablet:border-r tablet:border-theme-divider-tertiary',
+  'flex flex-col flex-1 px-8 tablet:pb-20 laptop:max-w-[40rem] tablet:border-r tablet:border-theme-divider-tertiary',
```

Comment:
Please make sure it doesn't break the article page

Reply:
Yes. These changes were actually intentional since there was a change in the actual layout.